dist/helpers.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+var truthTest = function truthTest(params, bodies, context, test) {
+  var key = params.key;
+  var val = params.val;
+  var main = bodies.main;
+
+  var elseBody = bodies["else"];
+  if (key && val) {
+    if (test(key, val)) {
+      if (main) {
+        return main(context);
+      }
+    } else if (elseBody) {
+      return elseBody(context);
+    }
+  }
+};
+
+var helpers = {
+  eq: function eq(context, params, bodies) {
```

Comment:
I have given this some for thought, and I think this is a good API. The `context` may or may not be a copy of the real context, but that doesn't change the API. The `params` and `bodies` are obviously necessary. Acting on the DOM (e.g. adding event listeners) is possible through access to the `bodies`.

The only other thing that helpers may need is some idea of nesting. I'm not sure whether that would come in the form of an existing param (perhaps `context`), a new param, or some global Tornado state variable. What do you think @jimmyhchan and @prashn64